roosevelt.js

```diff
@@ -138,6 +129,15 @@ module.exports = function(params) {
       require('./lib/jsCompiler')(app, mapRoutes);
     }
 
+     // initialize HTML validator
+    validateHTML();
+
+    function validateHTML() {
+      require('./lib/htmlValidator')(app, preprocessCss);
```

Comment:
In order for this sequence of events to be changed `validateHTML();` needs to be removed and replaced with `preProcessCss();`, moved up to line 120 and the callbacks in each function altered to represent the changed sequence.  (With `compileJs` calling `validateHTML` and `validateHTML` calling `mapRoutes`, etc)